### DISCUSSION TITLE
My opinion on Beta 1.3.0

### DISCUSSION THREAD

**Discussion by BroCrows** [WolfDen133/WFT-OLD#7] [General]
So today I’ll be giving my opinion on the BETA 1.3.0 update! 
First off great gob on the refresh timer!
Now lets get to the tags! Ill be scaling them on 1-10 (very useful(10) and very useless(1))

{tps} - 8
{load} - 7.5
{online_players} - 6
{max_players} - 3
{level} - 1

**Comment by WolfDen133** [comment]
As said in a previous discussion, there will be more in future.

**Reply by WolfDen133** [reply]
But your feedback is very important, and I will soon be implementing a addons system, much like ScoreHUD.

**Comment by kostamax27** [comment]
Hi, I would like you to add translations in different languages ​​for example:
Russian
French
Brazilian
I can help with translation into Russian
